test/server/cards/06-CotE/TacticiansCamp.spec.js

```diff
@@ -0,0 +1,30 @@
+describe('Tactician\'s Camp', function() {
+    integration(function() {
+        beforeEach(function() {
+            this.setupTest({
+                phase: 'conflict',
+                player1: {
+                    inPlay: ['akodo-toturi','matsu-berserker'],
+                    dynastyDiscard: ['tactician-s-camp']
+                },
+                player2: {
+                    inPlay: ['doji-whisperer']
+                }
+            });
+
+            this.camp = this.player1.placeCardInProvince('tactician-s-camp', 'province 1');
+            this.toturi = this.player1.findCardByName('akodo-toturi');
+            this.berserker = this.player1.findCardByName('matsu-berserker');
+
+            this.whisperer = this.player2.findCardByName('doji-whisperer');
+
+        });
+
+        it('should only give honored characters +1 MIL', function() {
+            this.toturi.honor();
+            expect(this.toturi.getMilitarySkill()).toBe(9);
```

Comment:
Shouldn't Toturi's mil skill be 10 here? I suspect this is another case of directly manipulating the game state meaning that the game state hasn't recalculated persistent effects

Reply:
Yes you're right. Added checkGameState.